src/web/app/pages-session/session-submission-page/session-submission-page.component.scss

```diff
@@ -0,0 +1,12 @@
+.card-plain {
+  background-color: #EAEFF5;
+  background-image: none;
+  border: 0;
+  -moz-box-shadow: 0 0 0 0;
+  -webkit-box-shadow: 0 0 0 0;
```

Comment:
Do we still need the above 2? The current front-end infrastructure comes with autoprefixer.